rules/sort-modules/compute-node-details.ts

```
@@ -0,0 +1,202 @@
+        shouldPartitionAfterNode = true
+        break
+      case AST_NODE_TYPES.TSEnumDeclaration:
+        selector = 'enum'
+        ;({ name } = nodeToParse.id)
+        dependencies = [...dependencies, ...extractDependencies(nodeToParse)]
+        break
+      case AST_NODE_TYPES.ClassDeclaration:
+        selector = 'class'
+        name = nodeToParse.id?.name
+        // eslint-disable-next-line no-case-declarations -- Easier to handle
+        let nodeDecorators = getNodeDecorators(nodeToParse)
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)